src/shared/cpu-set-util.c

```diff
@@ -290,3 +337,22 @@ int cpu_set_from_dbus(const uint8_t *bits, size_t size, CPUSet *set) {
         *set = TAKE_STRUCT(s);
         return 0;
 }
+
+int cpu_mask_add_all(CPUSet *mask) {
+        long m;
+
+        assert(mask);
+
+        m = sysconf(_SC_NPROCESSORS_ONLN);
+        assert_se(m > 0);
+
+        for (int i = 0; i < m; i++) {
+                int r;
```

Comment:
`int r` has the special privileged in our codebase to always be defined on outermost function scope. It's exempted from our usual rule that variables should always be declared at innermost scope. `int r` is the variable we almost always store function call error codes in, that's why it's a bit special to us. or in other words, please move this line 7 lines up, please